### PR TITLE
Add Italian holidays

### PR DESCRIPTION
**Pull request by mauricius**

### CODE REVIEW

**Review by gaedelprete — APPROVED** [review]
🇮🇹 👏 GJ @mauricius !

File: src/Countries/Italy.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+use Carbon\CarbonImmutable;
+
+class Italy extends Country
+{
+    public function countryCode(): string
+    {
+        return 'it';
+    }
+
+    /** @return array<string, CarbonImmutable> */
```
Comment (Nielsvanpach) [inline src/Countries/Italy.php:14]: This can be removed

**Review by Nielsvanpach — COMMENTED** [review]
Thanks! Could you rebase this branch, in order to run the tests here?

**Comment by Nielsvanpach** [comment]
Thanks!

**Pull request merged**